db/migrate/20200724150114_add_time_zone_and_service_region_to_changes_and_actions.rb

```diff
@@ -0,0 +1,9 @@
+class AddTimeZoneAndServiceRegionToChangesAndActions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :delay_henka_scheduled_changes, :time_zone, :string
+    add_column :delay_henka_scheduled_changes, :service_region_id, :integer
+
+    add_column :delay_henka_scheduled_actions, :time_zone, :string
+    add_column :delay_henka_scheduled_actions, :service_region_id, :integer
+  end
+end
```

Comment:
I was planning on not bumping the versioning until the actual changes to the models/workers went in.  Is there anything else you'd like to see here?